Formula/suil.rb

```diff
@@ -19,10 +21,16 @@ class Suil < Formula
 
   depends_on "pkg-config" => :build
   depends_on "gtk+"
+  depends_on "gtk+3"
   depends_on "lv2"
+  depends_on "qt@5"
+
+  patch :p1, :DATA
 
   def install
-    system "./waf", "configure", "--prefix=#{prefix}"
+    ENV.cxx11
+    system "./waf", "configure", "--prefix=#{prefix}", "--no-x11",
+        "--gtk2-lib-name=libgtk-quartz-2.0.0.dylib", "--gtk3-lib-name=libgtk-3.0.dylib"
```

Comment:
```suggestion
        "--gtk2-lib-name=#{shared_library("libgtk-quartz-2.0.0")}", "--gtk3-lib-name=#{shared_library("libgtk-3.0")}"
```

This should hopefully make it portable to Linux.